pkg/api/v1beta3/types.go

```diff
@@ -595,48 +593,48 @@ type PodStatusResult struct {
 // to hosts.
 type Pod struct {
 	TypeMeta   `json:",inline"`
-	ObjectMeta `json:"metadata,omitempty"`
+	ObjectMeta `json:"metadata,omitempty" description:"metadata specific to the Pod"`
 
 	// Spec defines the behavior of a pod.
-	Spec PodSpec `json:"spec,omitempty"`
+	Spec PodSpec `json:"spec,omitempty"description:"specification of the desired behavior of the pod"`
```

Comment:
Description is fine. Needs a space before "description".

Also add a link: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api-conventions.md#spec-and-status